### DISCUSSION TITLE
i want to start a desktop gui project to manage the seaweedfs configration

### DISCUSSION THREAD

**Discussion by fuyoo** [seaweedfs/seaweedfs#3262] [Ideas]
i want to start a desktop gui project to manage the seaweedfs configration

**Comment by fuyoo** [comment]
and listen the  service status

**Comment by chrislusf** [comment]
welcome!

**Reply by fuyoo** [reply]
if i do that, i am need to some idea such as:
1. does seaweedfs have an API to get the status API of master and nodes;
2. user's most care about things;

**Reply by fuyoo** [reply]
I'm not familiar with seaweedfs

**Reply by chrislusf** [reply]
there are gRPC APIs.

probably you need to get familiar with SeaweedFS first before the requirements/features come to you.

**Reply by fuyoo** [reply]
Ok, Thank you!
